libnetwork/drivers/windows/windows.go

```diff
@@ -109,16 +109,20 @@ const (
 	errNotFound = "HNS failed with error : The object identifier does not represent a valid object. "
 )
 
+var builtinLocalDrivers = map[string]struct{}{
+	"transparent": {},
+	"l2bridge":    {},
+	"l2tunnel":    {},
+	"nat":         {},
+	"internal":    {},
+	"private":     {},
+	"ics":         {},
+}
+
 // IsBuiltinLocalDriver validates if network-type is a builtin local-scoped driver
 func IsBuiltinLocalDriver(networkType string) bool {
-	if "l2bridge" == networkType || "l2tunnel" == networkType ||
-		"nat" == networkType || "ics" == networkType ||
```

Comment:
Very orthogonal, but "note to self" that I still need to look at "built-in" networks as a whole. ISTR there are some parts in the code, where the client checks if a driver is "pre-existing", but does so based on the _client_ code (hence, a Linux CLI connecting to a Windows daemon already makes assumptions based on Linux semantics).